docs/source/install.rst

```diff
@@ -68,7 +86,18 @@ variable)::
         'default': dj_database_url.parse('postgres:///your_database_name')
     }
 
-Now it's time to run migrations::
+If you prefer to use a simpler setup for your database you can use sqlite engine.
+To change database configuration to sqlite go to ``pydotorg/settings/local.py`` and:
```

Comment:
What do you think about converting this to mention the `DATABASE_URL` environment variable? Contributors can use SQLite without editing `pydotorg/settings/local.py`:

```sh
$ DATABASE_URL="sqlite:///pythondotorg.db" ./manage.py migrate
```

or

```sh
$ export DATABASE_URL="sqlite:///pythondotorg.db"
$ ./manage.py migrate
```